File: 7-troca de valores/troca.c
#include <stdio.h>

int main() {
    int a, b, temp;
    printf("Digite o valor de A: ");
    scanf("%d", &a);
    printf("Digite o valor de B: ");
    scanf("%d", &b);

    temp = a;
    a = b;
    b = temp;

    printf("Após a troca, A = %d e B = %d\n", a, b);
    return 0;
}

File: 7-troca de valores/troca.go
package main

import "fmt"

func main() {
	var a, b int
	fmt.Println("Digite o valor de A: ")
	fmt.Scan(&a)
	fmt.Println("Digite o valor de B: ")
	fmt.Scan(&b)

	a, b = b, a

	fmt.Printf("Após a troca, A = %d e B = %d\n", a, b)
}
